### ISSUE TITLE
SD calculation for standardized differences

### ISSUE THREAD

**Issue by mattcefalu** [mattcefalu/twang#19]
We currently calculate standardized differences using the marginal standard deviation of the characteristic across the full population. For example,

https://github.com/mattcefalu/twang/blob/cfdc5f8f416b79fbeaf12f900c11317757c30294/R/calcES.R#L30-L36

However, it may be more appropriate to use the pooled within treatment SD.

**Comment by ngreifer** [comment]
I agree that the pooled within-treatment SD would be preferable since that is what is recommended in the literature (e.g., Austin & Stuart, 2015, "Moving towards best practice when using inverse probability of treatment weighting (IPTW) using the propensity score to estimate causal treatment effects in observational studies") and what researchers think is being calculated.

**Comment by ngreifer** [comment]
I've actually changed my mind on this. We want balance to reflect the target population. The variance of the target population is the variance of entire unweighted sample, not the average of the variances in the two treatment groups. But the literature does not seem to agree with this.
